.github/workflows/tag_image_push.yml

```diff
@@ -53,6 +62,17 @@ jobs:
               exit 1
           fi
 
+      - name: Check if it is latest-lts release
+        if: github.event_name != 'workflow_dispatch'
+        run: |
+          IS_LATEST_LTS_FILE=.github/latest_lts
+          if [ -f IS_LATEST_LTS_FILE ]; then
+              echo "IS_LATEST_LTS=true" >> $GITHUB_ENV
+              echo "File '$IS_LATEST_LTS_FILE' exists. Assuming latest-lts release"
+          else
+              echo "File '$IS_LATEST_LTS_FILE' does not exist. Assuming NON-latest-lts release"
```

Comment:
May be set to `false` when not LTS like its done above for ENV.
Just for completeness

`echo "IS_LATEST_LTS=false" >> $GITHUB_ENV`

Reply:
I need to rework this, so I will postpone addressing this for now

Reply:
understood
I will leave it open for now